cypress/integration/page.spec.js

```diff
@@ -1,8 +1,43 @@
 'use strict';
 
-// TODO: edit it
-describe('Page', () => {
-  it('should be visitable', () => {
+Cypress.Commands.add('clickOnWall',
+  (position, xCoord, yCoord) => {
+    // NOTE: waiting for transition
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.get('.wall').click(position).wait(1000);
+
+    cy.get('.spider').then(($spider) => {
+      const spider = $spider.get(0);
+      const spiderTop = spider.offsetTop;
+      const spiderLeft = spider.offsetLeft;
+
+      expect(spiderTop).to.equal(xCoord);
+      expect(spiderLeft).to.equal(yCoord);
+    });
+  });
+
+describe('Spider', () => {
+  beforeEach('Open site', () => {
```

Comment:
where is the message 'Open site' displayed?

Reply:
Nowhere. That’s just for me
can remove